lib/middleware.js

```diff
@@ -512,6 +516,22 @@ module.exports = function middlewareConstructor(env) {
           res.json({"status": 200});
         });
       };
+    },
+
+    /**
+     * If there's an oauth token, decrypt it and set the
+     * local user object.
+     */
+    maybeDecryptToken: function(req, res, next) {
```

Comment:
Although you may not be decrypting in this case, prefixing it with maybe seems like an incorrect convention.